### PR TITLE
10952 website high contrast

### PR DESCRIPTION
**Pull request by jdhuntington**
#### Pull request checklist

- [X] Addresses an existing issue: #10952 (Issues 2-5)
- [X] Include a change request file using `$ yarn change`

#### Description of changes

- Fix the left-hand navigation by omitting the gradient background
- Fix several buttons by fielding the high contrast selector

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/11070)

### CODE REVIEW

File: apps/fabric-website/src/styles/_base.scss [hunk 1]
```diff
@@ -62,8 +73,14 @@
   }
 
   // Feedback section styles
-  .FeedbackList-button .ms-Button-label { // Feedback button color overridden by UHF
+  .FeedbackList-button .ms-Button-label {
+    // Feedback button color overridden by UHF
     @include ms-fontColor-white;
+
+    @include high-contrast {
+      // The gradient doesn't behave well in high contrast, so get rid of it.
```
Comment (ecraig12345) [inline apps/fabric-website/src/styles/_base.scss]: Different comment?

Comment (jdhuntington) [reply apps/fabric-website/src/styles/_base.scss]: good catch

File: apps/fabric-website/src/styles/_base.scss [hunk 2]
```diff
@@ -62,8 +73,13 @@
   }
 
   // Feedback section styles
-  .FeedbackList-button .ms-Button-label { // Feedback button color overridden by UHF
+  .FeedbackList-button .ms-Button-label {
+    // Feedback button color overridden by UHF
     @include ms-fontColor-white;
+
+    @include high-contrast {
+      color: window;
```
Comment (maxwellred) [inline apps/fabric-website/src/styles/_base.scss]: @jdhuntington This is potentially super-nit but aren't system colors usually pascal case? If this doesn't cause an issue then disregard.

Comment (jdhuntington) [reply apps/fabric-website/src/styles/_base.scss]: Good call; that does appear to be correct. (TIL!)

**Pull request merged**